x/evm/types/key.go

```diff
@@ -23,7 +23,6 @@ const (
 var (
 	KeyPrefixBlockHash   = []byte{0x01}
 	KeyPrefixBloom       = []byte{0x02}
-	KeyPrefixLogs        = []byte{0x03}
 	KeyPrefixCode        = []byte{0x04}
```

Comment:
KeyPrefixCode should replace []byte{0x03} if an exporting-upgrade is on schedule and all latter ones should be kept in order.